### ISSUE TITLE
Closed item

### ISSUE THREAD

**Issue by Parikshitkale13** [feldroy/django-crash-course#284]
Deleted abusive and obscenity-laced description from person who had pirated copy of book. We had no record of their purchase. As we don't tolerate abuse in our open source projects, we will not tolerate abuse in our commercial efforts either.

-- @pydanny

**Comment by LeonT-A** [comment]
I agree with you, though I will admit I didn't give this book a fair shot. I only made it about 1/4 of the way through before I hopped ship.

The length vs content just doesn't add up. Way too much fluff in the form of line spacing, images, and general layout. This book could've literally been half the size. Getting a solid education seemed impossible, but they claimed otherwise. Then when they had you pull an entire repo, that was the deal-breaker. No thanks, I don't wanna copy-paste the entire setup, I want to follow along and create it so I know how it works.

Again, I didn't complete the book, maybe it brings everything around, and I'm sure I would've learned some neat tricks because the author did feel knowledgeable. However, based on previous books and their content quality, this felt like it was going to be a waste of time.
